e2e/scripts/run.sh

```diff
@@ -2,37 +2,59 @@
 set -eu
 set -o pipefail
 
+function print_logs {
+	docker logs ${CONTAINER_SERVER}1
+	docker logs ${CONTAINER_SERVER}2
+	docker logs ${CONTAINER_PROXY}
+}
+
+trap print_logs EXIT
+
+mkdir -p "${WORKSPACE}/results"
+
+# Generate sysadmin
+echo "Generating sysadmin ..."
+docker exec \
+	${CONTAINER_SERVER}1 \
```

Comment:
QQ: did I miss the setup for `${CONTAINER_SERVER}2`?

Reply:
We only need to create it once per cluster as the DB is shared.